request-manager/grapher/grapher.go

```diff
@@ -630,19 +630,20 @@ func (o *Grapher) remapNodeArgs(n *NodeSpec, args map[string]interface{}) (map[s
 }
 
 // Given a node definition and two args sets. Copy the arguments that
-// are defined in the "sets" clause into the main args map.
+// are defined in the "sets/arg" clause into the main args map under
+// the name defined by the "sets/as" field.
 func (o *Grapher) setNodeArgs(n *NodeSpec, argsTo, argsFrom map[string]interface{}) error {
 	if len(n.Sets) == 0 {
 		return nil
 	}
 	for _, key := range n.Sets {
 		var ok bool
 		var val interface{}
-		val, ok = argsFrom[key]
+		val, ok = argsFrom[key.Arg]
 		if !ok {
-			return fmt.Errorf("expected %s to set %s in jobargs", n.NodeType, key)
+			return fmt.Errorf("expected %s to set %s as %s in jobargs", n.NodeType, key.Arg, key.As)
```

Comment:
I would omit the `as %s` part of the error . If I see `expected job/type to set argA as argB in jobargs`, it's not clear to me whether the job was supposed to set `argA` or `argB`.